flashinfer/gemm.py

```diff
@@ -1939,15 +1967,16 @@ def mm_fp4(
             expanded_b_descale_shape,
             expanded_b_descale_stride,
             cudnn.data_type.FP4_E2M1,
-            torch.float8_e4m3fn,
+            torch.float8_e4m3fn if use_nvfp4 else torch.float8_e8m0fnu,
```

Comment:
Also note that `torch.float8_e8m0fnu` is introduced in torch 2.8 and we should care about the case of torch 2.7 before it's end-of-life.